src/psaform/psaform.jsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { Grid, Typography, Snackbar, Checkbox, FormGroup, FormControlLabel } from "@mui/material";
+import PSAButton from "../button";
+import PSATextField from "../Textfield";
+
+export const PSAForm = ({
+  apiUrl,
+  submitMessage,
+  headerTitle,
+  fields, // Single prop for both text fields and checkboxes
+  buttons,
+  consentRedux,
+  pirschAnalytics,
```

Comment:
remove `consentRedux` and `pirschAnalytics` and the `useEffect` in the code as well